contracts/SmartLoansFactory.sol

```diff
@@ -36,36 +39,42 @@ contract SmartLoansFactory is IBorrowersRegistry {
     pool = _pool;
     priceProvider = _priceProvider;
     assetsExchange = _assetsExchange;
+    SmartLoan smartLoanImplementation = new SmartLoan();
+    upgradeableBeacon = new UpgradeableBeacon(address(smartLoanImplementation));
+    upgradeableBeacon.transferOwnership(msg.sender);
   }
 
   function createLoan() public returns(SmartLoan) {
```

Comment:
I think we should add the validation checking if msg.sender doesn't have any smartLoans already as we discussed.
(Here and in createAndFund)